redbeams/src/main/java/com/sequenceiq/redbeams/controller/mapper/NotFoundExceptionMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.redbeams.controller.mapper;
```

Comment:
This conversion might also be moved to the common.

Reply:
It hasn't been moved yet. I'm still not sure whether we are keeping the common module long term.